lib/datapackage.py

```diff
@@ -27,30 +27,30 @@ def create_json():
         if script_exists:
             print("Dataset already available. Check the list or try a different shortname")
 
-    contents['title'] = input("Title/Name: ")
-    contents['description'] = input("Description: ")
-    contents['citation'] = input("Citation: ")
-    contents['homepage'] = input("Site/Homepage of dataset: ")
+    contents['title'] = input("Title/Name: ").strip()
+    contents['description'] = input("Description: ").strip()
+    contents['citation'] = input("Citation: ").strip()
+    contents['homepage'] = input("Site/Homepage of dataset: ").strip()
     tags = input("Tags (separated by ';'): ").split(';')
     contents['keywords'] = [tag.strip() for tag in tags if tag.strip() != ""]
     contents['resources'] = []
 
     # Add tables -
     while True:
-        addTable = input("\nAdd Table? (y/N): ")
-        if addTable.strip().lower() in ["n", "no"]:
+        addTable = input("\nAdd Table? (y/N): ").strip()
+        if addTable.lower() in ["n", "no"]:
             break
-        elif addTable.strip().lower() not in ["y", "yes"]:
+        elif addTable.lower() not in ["y", "yes"]:
             print("Not a valid option\n")
             continue
         else:
             table = {}
-            table['name'] = input("Table name: ")
-            table['url'] = input("Table URL: ")
+            table['name'] = input("Table name: ").strip()
+            table['url'] = input("Table URL: ").strip()
             table['dialect'] = {}
 
-            d_opts = ['nulls (separated by \';\')',  # list of str
-                      'replace_columns (separated by \';\')',  # list of tuples
+            d_opts = ['replace_columns (separated by \';\', with comma-separated values)',  # list of tuples
```

Comment:
I think these are defaults for all scripts/tables and we do not need  read/initialize them  them all the time, So we can take them out of the loop.